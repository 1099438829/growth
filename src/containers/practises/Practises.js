import React, { Component } from 'react';
import { ScrollView } from 'react-native';
import { Actions } from 'react-native-router-flux';
import { ListItem, List } from 'react-native-elements';

class Practises extends Component {
  static componentName = 'Practises';

  render() {
    return (
      <ScrollView>
        <List>
          <ListItem
            title={'LeetCode'}
          />
          <ListItem
            title={'算法'}
            onPress={() => Actions.algorithmView()}
          />
          <ListItem
            title={'数据结构'}
          />
          <ListItem
            title={'设计模式'}
          />
        </List>
      </ScrollView>
    );
  }
}

export default Practises;
