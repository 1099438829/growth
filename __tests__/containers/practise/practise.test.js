import 'react-native';
import React from 'react';

import renderer from 'react-test-renderer';
import Practises from '../../../src/containers/practises/Practises';

describe('test practise', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Practises />,
    );

    const treeJson = tree.toJSON();
    expect(treeJson).toMatchSnapshot();
  });
});
