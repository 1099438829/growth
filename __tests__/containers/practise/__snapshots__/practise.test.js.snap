// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test practise renders correctly 1`] = `null`;
